test/features/step_definitions/addPupilSteps.js

```diff
@@ -0,0 +1,15 @@
+var {defineSupportCode} = require('cucumber');
```

Comment:
code style needs to align with standardJS also use const/let instead of var accordingly